ejp_xml_pipeline/etl.py

```diff
@@ -172,18 +175,19 @@ def download_load2bq_cleanup_temp_files(
                     s3_object
                 )
                 if written_file_row_count > batch_size_limit:
-                    named_temp_file.flush()
+                    writer.flush()
                     load_and_delete_temp_objects(
                         gcp_project, dataset,
-                        bq_table, tempfile_name,
+                        bq_table, temp_file_name,
                         s3_bucket, s3_objects_written_to_file
                     )
-                    named_temp_file.truncate()
+                    writer.truncate()
                     s3_objects_written_to_file = []
                     written_file_row_count = 0
+            writer.flush()
```

Comment:
I guess this `flush` is what was causing it?

Reply:
both flush, and NamedTemporaryFile